airflow/models.py

```
@@ -811,6 +843,14 @@ def get_hook(self):
         except Exception:
             pass
 
+    def get_kms_hook(self):
+        try:
+            if self.conn_type == 'google_cloud_platform':
+                from airflow.contrib.hooks.gcp_kms_hook import GoogleCloudKMSHook
+                return GoogleCloudKMSHook(gcp_conn_id=self.conn_id)
+        except Exception:
+            pass
```

If we're going to ignore the exception, we should definitely at least log it.